PROVESFlightControllerReference/test/int/bridge_uplink_test.py

```
@@ -0,0 +1,471 @@
+    import time as _t
+
+    _t.sleep(8)
+
+
+def test_spray_seq(fprime_test_api: IntegrationTestAPI):
+    """Spray many bypass GET_SEQ_NUM rapidly to land one in a flight RX window."""
+    import time as _t
+
+    n = int(os.environ.get("SPRAY_N", "20"))
+    gap = float(os.environ.get("SPRAY_GAP", "0.3"))
+    for i in range(n):
```

Ruff (0.15.20)

[warning] Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)